[Compression] Update backfill-procedure support (#1257)
timescale/docs: timescaledb/how-to-guides/compression/backfill-historical-data.md

# Backfill historical data on compressed chunks

In the [TimescaleDB extras][timescaledb-extras] GitHub repository, we provide
explicit functions for
[backfilling batch data to compressed chunks][timescaledb-extras-backfill],
which is useful for inserting a *batch* of backfilled data (as opposed to

> individual row inserts). By "backfill", we mean inserting data corresponding to
> a timestamp well in the past, which given its timestamp, already corresponds to
> a compressed chunk.

Reviewer: ```suggestion
individual row inserts). When you backfill data, you insert data corresponding to
a timestamp well in the past, which has a timestamp that already corresponds to
a compressed chunk.
```

In the below example, we backfill data into a temporary table; such temporary
tables are short-lived and only exist for the duration of the database
session. Alternatively, if backfill is common, one might use a normal table for
this instead, which would allow multiple writers to insert into the table at
the same time before the `decompress_backfill` process.

<procedure>

## Backfilling using the `decompress_backfill` procedure

<highlight type="important">
The `decompress_backfill` function doesn't support distributed hypertables. To

> backfill a distributed hypertable, manually decompress the chunks.

Reviewer 2: By adding "...before the data is inserted." to the end of the sentence, the user might be able to understand it better.

> backfill a distributed hypertable, manually decompress the chunks.

Author: ```suggestion
backfill a distributed hypertable, manually decompress the chunks before inserting data.
```

 CALL decompress_backfill(staging_table=>'cpu_temp', destination_hypertable=>'cpu');`
 ```

</procedure>

> If using a temp table, the table is automatically dropped at the end of your

Reviewer: ```suggestion
If you are using a temporary table, the table is automatically dropped at the end of your
```

> database session. If using a normal table, after you are done backfilling the

Reviewer: ```suggestion
database session. If you are using a regular table, after you have backfilled the
```

CALL run_job(<job_id>);
```

> ## Future Work

Reviewer: ```suggestion
## Future work
```
